experimental/dds/tree2/src/core/revertible/revertible.ts

```diff
@@ -12,6 +12,9 @@
 export interface Revertible {
 	/** Indicates the type of edit that produced this revertible. */
 	readonly kind: RevertibleKind;
+	/**
+	 * Information about which client created the edit.
+	 */
 	readonly origin: {
 		readonly isLocal: boolean;
```

Comment:
Docs for this too? 😁

Reply:
API Extractor doesn't believe in such docs, so I'm guessing API-Documenter won't consume them? Should probably document it anyway.

Reply:
```suggestion
		/**
		 * Indicates if the {@link Revertible} is from the local client (true) or a remote client (false).
		 */
		readonly isLocal: boolean;
```